### DISCUSSION TITLE
ClearIndicator is hidden when Select "isDisabled" is true and isMulti. How to keep clear indicator visisble when select is disabled?

### DISCUSSION THREAD

**Discussion by vinesh20** [JedWatson/react-select#5316] [Q&A]
We are having scenario where we have to keep showing the ClearIndicator when 
isMulit is true and
isDisabled is true

Rightnow, when i set isDisabled to true, clear indicator is removed from the DOM

Please help me if there is that i can do keep it visible.

**Comment by ishaanbedi** [comment]
Hey @vinesh20, I know this is super late, but do you recall a solution to this?

**Comment by brasillero** [comment]
Same problem
